packages/backend/vitest.setup.mts

```diff
@@ -0,0 +1,14 @@
+import { beforeAll, afterEach, afterAll } from 'vitest';
+import { server } from './src/mock-server';
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+server.events.on('request:start', ({ request }) => {
+  // FOR DEBUGGING
+  // console.log('MSW intercepted:', request.method, request.url);
```

Comment:
👀 want to leave this? if yes, maybe we put it behind an env var or something?

Reply:
@BRKalow I am fine either way. My thought was leaving it commented out if devs wanted to debug this in the future, but an ENV might be better. Do we have a naming standard for ENV vars?

Reply:
We do not. `DEBUG_MOCK_REQUESTS` maybe? And add to the README 👀

Reply:
Addressed in [`ae16e84` (#4296)](https://github.com/clerk/javascript/pull/4296/commits/ae16e84a53e7109bf7772fb25d484dba09e30f73)